kratos/KratosMultiphysics.json

```diff
@@ -1,6 +1,6 @@
 {
 	"wheel_name": "KratosMultiphysics",
-	"included_modules": [],
+	"included_modules": ["response_functions"],
```

Comment:
There are also two other folders at this level: "testing" and "application_generator" but from my understanding they need not to be part of the wheel.